socorro/external/boto/crash_data.py

```diff
@@ -95,7 +95,7 @@ def get(self, **kwargs):
             raise MissingArgumentError("uuid")
 
         try:
-            return self.get_unredacted_processed(params.uuid)
+            return self.get_unredacted_crash_report(params.uuid)
```

Comment:
I'm going to undo this and related changes to the `TelemetryBotoS3CrashStorage`.